### DISCUSSION TITLE
[Ground Items] Add option to hide despawn timers for other player's loot (restore old functionality)

### DISCUSSION THREAD

**Discussion by mmmilord** [runelite/runelite#17756] [Ideas]
> Ground items despawn timers now use the enhanced client despawn times, which works more reliably and on all items

https://runelite.net/blog/show/2024-04-17-1.10.27-Release/

Unless I'm mistaken, prior to this change, timers only showed for your own loot. Now they appear on drops from players. I very much preferred how it displayed before, if there is a way to only keep timers on my drops, it's very disorientating for me.

**Comment by Alexsuperfly** [comment]
the Only show loot option

**Reply by mmmilord** [reply]
But this hides everything, that's not how it worked pre update. I want to see other's loot, just not their despawn timers, I only care about when my own loot despawns.

e.g. before:

You kill a dragon, 
Dragon bones - 180s 

Someone else kills a dragon,
Dragon bones

**Comment by EtubeOSRS** [comment]
Came here specifically to make this suggestion as well. It's extremely useful on an Ironman to be able to use the drop timers to distinguish loot that I can pick up from loot that I cannot pick up, but I still want to be able to see others' stuff on the ground.

Additionally, even if I check "Only show loot," if I come away from an item I dropped then come back, the ground item text will go away even though RL is still tracking the despawn timer if I check the option back off.
